features/FxA-54-connect-another-device-phase-3/README.md

```diff
@@ -0,0 +1,367 @@
+# Connect another device, phase 3: deep link with pre-filled email address
+
+We want to further reduce barriers
+to connecting another device after account registration,
+by making the Firefox installation link that is sent to users via SMS
+open the browser on the FxA sign-in page
+with their email address pre-filled.
+
+* [Ubiquitous language](#ubiquitous-language)
+* [Hypothesis](#hypothesis)
+* [Metrics](#metrics)
+* [User flow](#user-flow)
+* [Implementation](#implementation)
+  * [Content server](#content-server)
+  * [Auth server](#auth-server)
+  * [Auth database](#auth-database)
+  * [Mobile clients](#mobile-clients)
+  * [Mock-ups](#mock-ups)
+* [Test plan/acceptance criteria](#test-planacceptance-criteria)
+* [Open questions](#open-questions)
+
+## Ubiquitous language
+
+* Connect another device:
+  The feature whereby,
+  after creating an account,
+  users are ushered through an expedited flow
+  to install Firefox on a mobile device
+  and connect it to Sync.
+
+* Deep link:
+  An app store link for installing Firefox on a mobile device
+  and connecting it to Sync.
+  The link redirects through Adjust,
+  which is a third-party service that handles
+  all of the redirection magic.
+
+* signinCode:
+  A single-use, time-limited, URL-safe, base64-encoded 8-byte code,
+  which can be exchanged in return for a user's primary email address.
+
+## Hypothesis
+
+More users will sign in to Sync on their mobile device
+if we pre-fill the sign-in form with their email address.
+
+## Metrics
+
+We will know our hypothesis to be true
+if the percentage of completed flows continued from treatment group flows
+is greater than the percentage of completed flows continued from control group flows.
+This is measured by the following redash query:
+
+* https://sql.telemetry.mozilla.org/queries/4808
+
+As a secondary signal, we should also see an increase
+in the percentage of Sync users who become multi-device
+within 2 days of creating an account:
+
+* https://sql.telemetry.mozilla.org/queries/5469
+
+These metrics and any others that are relevant
+will be tracked by charts on the following dashboard:
+
+* https://sql.telemetry.mozilla.org/dashboard/fxa-connect-another-device-phase-3
+
+## User flow
```

Comment:
You should include a brief description of the current flow so that the control is known and it is more clear how success is measured.